pymc/model/core.py

```diff
@@ -1584,6 +1584,36 @@ def __getitem__(self, key):
     def __contains__(self, key):
         return key in self.named_vars or self.name_for(key) in self.named_vars
 
+    def __copy__(self):
```

Comment:
Mention `clone_model` / add code example on how to access variables after cloning `new_model["var_name"]` since the variables are copied and the original ones are not part of the cloned model

Reply:
Okay, I will mention the `clone_model` documentation and I will add a code example for accessing the variables after cloning!